server/src/instant/lib/ring/websocket.clj

```diff
@@ -79,8 +79,10 @@
       (if (> ms-since-last-message idle-timeout-ms)
         (tracer/with-span! {:name "socket/close-inactive"}
           (IoUtils/safeClose channel))
-        (WebSockets/sendPingBlocking (ByteBuffer/allocate 0)
-                                     channel)))
+        (try (WebSockets/sendPingBlocking
+              (ByteBuffer/allocate 0)
+              channel)
+             (catch java.io.IOException _))))
```

Comment:
Can we just catch the closedchannelexception instead of all io exceptions?

Reply:
good idea! Will do this

Reply:
cc @dwwoelfel 

It looks like this can throw either a `ClosedChannelException`, _or_ a `IOException` with the message ['Channel is closed'](https://github.com/undertow-io/undertow/blob/main/core/src/main/java/io/undertow/websockets/core/WebSocketChannel.java#L353-L355).

I went ahead and updated the code to specifically catch: ClosedChannelException, and IOException with this message.